lighthouse-core/audits/third-party-summary.js

```diff
@@ -15,7 +15,7 @@ const MainThreadTasks = require('../computed/main-thread-tasks.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides details about the code on a web page that the user doesn't control (referred to as "third-party code"). This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Third-Party usage',
+  title: 'Limit third-party usage',
```

Comment:
only remaining thing is @paulirish suggested `Minimize third-party usage` as an alternative verb here